plugins/devtools-backend/config.d.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  /**
+   * DevTools configuration.
+   */
+  devTools?: {
+    /**
+     * External dependency configuration.
+     */
+    externalDependencies?: {
+      /**
+       * The list of endpoints to check.
+       */
+      endpoints?: Array<{
```

Comment:
this is neat; sort of a lightweight way of doing this.

overtime, allowing this to extend out to things like StatusPage and other services which may exist at an enterprise would be good. ( #futurework)

Reply:
Yes, that seems better to me, I've adjusted this 👍